features/step_definitions/json_api_average.rb

```diff
@@ -87,3 +87,21 @@
     expect(item['user_id']).to eq 2
   end
 end
+
+Given(/^a set of ratings for several ratable types$/) do
+  3.times do |i|
+    4.times { |t| create :rating, ratable_type: "thing_type_#{ i }", user_id: 2 }
```

Comment:
I'd remove the `|t|` param here, as you aren't using it. For future reference, the usual single char secondary options for iterators are `j`, `k`, and `n`.